lib/matplotlib/_text_helpers.py

```diff
@@ -65,7 +65,7 @@ def layout(string, font, *, kern_mode=Kerning.DEFAULT):
     """
     x = 0
     prev_glyph_idx = None
-    char_to_font = font._get_fontmap(string)
+    char_to_font = font._get_fontmap(string)  # TODO: Pass in language.
```

Comment:
Note, this function is getting rewritten with libraqm, so this is just a note for later.